### PR TITLE
Do not delay sign EventFlow assemblies

### PR DESCRIPTION
**Pull request by karolz-ms**
This change is intended to address [issue 180](https://github.com/Azure/diagnostics-eventflow/issues/180) I ended up with just removing the strong name signing altogether. It causes problems at dev time and on non-Windows platforms and I am not sure the cost is justified. Note that the assemblies in Nuget packages will still be signed as part of release build.

### CODE REVIEW

**Comment by SergeyKanzhelev** [comment]
https://github.com/Microsoft/ApplicationInsights-dotnet/blob/beaa444d435e806af7456862f9f3ba9668f6fcfe/Signing.targets#L41

**Comment by SergeyKanzhelev** [comment]
https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/public-signing.md

**Comment by karolz-ms** [comment]
Thanks, @SergeyKanzhelev! good to know 

After reading through the materials you shared I much more convinced that getting rid of strong name signing altogether is the right approach for EventFlow.

@xiaomi7732 a review would be appreciated :-)

File: src/Microsoft.Diagnostics.EventFlow.Core/Implementations/DiagnosticPipelineFactory.cs [hunk 1]
```diff
@@ -316,25 +316,25 @@ private static void ReportInvalidPipelineConfiguration(IHealthReporter healthRep
             Debug.Assert(healthReporter != null);
 
             inputFactories = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
-            inputFactories["EventSource"] = "Microsoft.Diagnostics.EventFlow.Inputs.EventSourceInputFactory, Microsoft.Diagnostics.EventFlow.Inputs.EventSource, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a";
```
Comment (xiaomi7732) [inline src/Microsoft.Diagnostics.EventFlow.Core/Implementations/DiagnosticPipelineFactory.cs]: These looks like real-signed public token. I believe these are going to be used in the reflector for loading. Are these types still going to be loaded in signed bits in run-time?

File: src/Microsoft.Diagnostics.EventFlow.FilterParserGenerator/FilterParser.peg [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ﻿@namespace Microsoft.Diagnostics.EventFlow
 @classname FilterParser
-@accessibility internal
```
Comment (xiaomi7732) [inline src/Microsoft.Diagnostics.EventFlow.FilterParserGenerator/FilterParser.peg]: Why expose this as public now?

File: src/Microsoft.Diagnostics.EventFlow.Core/Implementations/DiagnosticPipelineFactory.cs (hunk 1 above)
Comment (karolz-ms) [reply src/Microsoft.Diagnostics.EventFlow.Core/Implementations/DiagnosticPipelineFactory.cs]: Yes. The signing that remains is Authenticode signing that confirms the DLL was build by Microsoft; it does not affect the .NET assembly loader.

Comment (xiaomi7732) [reply src/Microsoft.Diagnostics.EventFlow.Core/Implementations/DiagnosticPipelineFactory.cs]: Okay. I don't think I understand the auth-code process but I'll trust your words on this. 💯

File: src/Microsoft.Diagnostics.EventFlow.FilterParserGenerator/FilterParser.peg (hunk 2 above)
Comment (karolz-ms) [reply src/Microsoft.Diagnostics.EventFlow.FilterParserGenerator/FilterParser.peg]: Good catch, I think it was a leftover from some earlier experiments, but I tried again and as far as I can tell, this change is not necessary. I will revert it after pushing this PR.

**Pull request merged**